src/instructlab/model/verify.py

```diff
@@ -0,0 +1,65 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# Standard
+from pathlib import Path
+
+# Third Party
+from sigstore.errors import VerificationError
+import click
+
+# First Party
+from instructlab import configuration as config
+from instructlab import signing, utils
+
+
+@click.command()
+@click.option(
+    "--model-path",
+    type=click.Path(),
+    default=config.DEFAULT_MODEL_PATH,
+    show_default=True,
+    help="Path to the model to be signed.",
+)
+@click.option(
+    "--bundle-path",
+    type=click.Path(),
+    default=None,
+    show_default=True,
+    help="Path to save the Sigstore bundle file after signing.",
+)
+@click.option(
+    "--identity",
+    help="Certificate identity to verify against."
+    "Typically an email provided by the OIDC identity provider."
+    "Example: a model signed using a Github account under hello@instructlab.ai would use hello@instructlab.ai.",
+)
+@click.option(
+    "--issuer",
+    help="Certificate identity's issuing authority.",
+    default="https://github.com/login/oauth",
+    show_default=True,
+)
```

Comment:
Another option could be to pick up a .yaml file here that contains the identity and issuer and is provided by the signer ( ideally this file would also be signed) and its contents are displayed when the signature verification is done so that the user can see whether the signature expectations are met correctly.
```
oidc:
  -identity: user@gmail.com
  -issuer: https://accounts.google.com
```